$OpenBSD: patch-data_org_gnome_seahorse_recipients_convert,v 1.1.1.1 2011/09/18 20:10:59 ajacoutot Exp $

GLib-GIO-ERROR **: Settings schema 'org.gnome.seahorse.recipients' does not contain a key named 'show-validity'

--- data/org.gnome.seahorse.recipients.convert.orig	Sun Sep 18 14:27:07 2011
+++ data/org.gnome.seahorse.recipients.convert	Sun Sep 18 14:28:23 2011
@@ -1,6 +1,2 @@
 [org.gnome.seahorse.recipients]
-show-validity = /apps/seahorse/listing/show_validity
-show-expiry = /apps/seahorse/listing/show_expires
-show-trust = /apps/seahorse/listing/show_trust
-show-type = /apps/seahorse/listing/show_type
-sort-by = /apps/seahorse/listing/sort_by
\ No newline at end of file
+sort-by = /apps/seahorse/listing/sort_by
